labels.json

```diff
@@ -218,5 +218,15 @@
     "name": "type: security",
     "description": "Issue related to Security",
     "color": "3E83FC"
+  },
+  {
+    "name": "type: chore",
+    "description": null,
+    "color": "006b75"
+  },
+  {
+    "name": "rfc: discussion",
```

Comment:
```suggestion
    "name": "type: rfc",
```

Reply:
also `type: discussion`?

Reply:
@jeremyw24 to Caspian's point - what's the intent of these? [RFC's](https://en.wikipedia.org/wiki/Request_for_Comments) and discussions are seperate things.

I'll add some support for renaming labels to this automation, but that's not there atm so simpler to get this right first go.

Reply:
@kimburgess there is some human error involved here ;) 
it should be type: discussion == an issue resolved via discussion.
@caspiano requested type: rfc but I messed it up when I created it.

Reply:
added commit with the correction now.